# Generated by Django 3.1 on 2020-09-26 12:36

from django.conf import settings
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('managegoal', '0003_auto_20200926_1218'),
    ]

    operations = [
        migrations.AlterField(
            model_name='changeapplication',
            name='good',
            field=models.IntegerField(default=0),
        ),
        migrations.AlterField(
            model_name='goal',
            name='endtime',
            field=models.TimeField(blank=True, null=True),
        ),
        migrations.AlterField(
            model_name='goal',
            name='explanation',
            field=models.TextField(blank=True, max_length=1000, null=True),
        ),
        migrations.AlterField(
            model_name='goal',
            name='image',
            field=models.ImageField(blank=True, null=True, upload_to=''),
        ),
        migrations.AlterField(
            model_name='goal',
            name='otherinfo',
            field=models.TextField(blank=True, max_length=1000, null=True),
        ),
        migrations.AlterField(
            model_name='goal',
            name='starttime',
            field=models.TimeField(blank=True, null=True),
        ),
        migrations.AlterField(
            model_name='goal',
            name='url',
            field=models.URLField(blank=True, null=True),
        ),
        migrations.AlterField(
            model_name='impression',
            name='good',
            field=models.IntegerField(default=0),
        ),
        migrations.AlterField(
            model_name='impression',
            name='good_users',
            field=models.ManyToManyField(blank=True, null=True, related_name='good_users', to=settings.AUTH_USER_MODEL),
        ),
    ]
